javascript/javascript2/week2/doublingOfNumbers.js

```diff
@@ -0,0 +1,13 @@
+let numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+let isOdd = (numbers) => isOdd(numbers) === 'NaN' ? isOdd(numbers) : !isOdd(numbers);
```

Comment:
You probably tried to achieve this:

```suggestion
const isOdd = (number) => number % 2 !== 0;
const oddNumbers = numbers.filter(isOdd);
```

Reply:
yeah i think